sdks/DeviceSDK.ts

```diff
@@ -21,6 +21,8 @@ export default class DeviceSDK {
     @Inject private databaseSdk: DataBaseSDK;
     private config: IConfig;
     private apiKey: string;
+    private v1_api_path = "/api/api-manager/v1/consumer/desktop_device/credential/register";
```

Comment:
rename it to **deviceRegistryV1APIPath** and **deviceRegistryV2APIPath**

Reply:
done